pwnlib/commandline/libcdb.py

```diff
@@ -196,25 +217,48 @@ def main(args):
         if len(pairs) % 2 != 0:
             log.failure('Uneven number of arguments. Please provide "symbol offset" pairs')
             return
-        
+
         symbols = {pairs[i]:pairs[i+1] for i in range(0, len(pairs), 2)}
-        matched_libcs = find_libc({'symbols': symbols})
+        matched_libcs = libcdb.search_by_symbol_offsets(symbols, raw=True, unstrip=args.unstrip, offline=args.offline)
         for libc in matched_libcs:
-            handle_remote_libc(args, libc)
+            print_libc(libc)
+            fetch_libc(args, libc)
 
     elif args.libc_command == 'hash':
         for hash_value in args.hash_value:
-            matched_libcs = find_libc({args.hash_type: hash_value})
-            for libc in matched_libcs:
-                handle_remote_libc(args, libc)
+            libs_id = None
+            hash_type = args.hash_type if args.hash_type != "buildid" else "build_id"
+
+            # Search and download libc
+            exe_path = libcdb.search_by_hash(hash_value, hash_type, args.unstrip, args.offline)
+
+            if exe_path:
+                libs_id = read(exe_path + ".id").decode()
+
+            if not libs_id:
```

Comment:
Since the gitlab/libcdb provider doesn't know about that id, this (and similar checks) breaks lookup using that repo or am I missing something?